final_prep/mod_zero_hero.rb

```diff
@@ -1,84 +1,154 @@
 #   - If the danger_level is below 10, it means it is not worth your time and should result in printing the string "Meh. Hard pass." to the terminal.
 
+def assess_situation(danger_level, save_the_day, bad_excuse)
+  if danger_level > 50
+    puts bad_excuse
+  elsif danger_level >= 10 && danger_level <= 50
+    puts save_the_day
+  else
+    puts "Meh, hard pass."
+  end
+end
 #Test Cases
 announcement = 'Never fear, the Courageous Curly Bracket is here!'
 excuse = 'I think I forgot to lock up my 1992 Toyota Coralla. Be right back.'
 # assess_situation(99, announcement, excuse) > Should print - 'I think I forgot to lock up my 1992 Toyota Coralla. Be right back.'
-#assess_situation(21, announcement, excuse) > should print - 'Never fear, the Courageous Curly Bracket is here!'
-#assess_situation(3, announcement, excuse) > should print - "Meh. Hard pass."
-
+# assess_situation(21, announcement, excuse) > should print - 'Never fear, the Courageous Curly Bracket is here!'
+# assess_situation(3, announcement, excuse) > should print - "Meh. Hard pass."
+assess_situation(99, announcement, excuse)
+assess_situation(21, announcement, excuse)
+assess_situation(3, announcement, excuse)
 # Declare a new variable - scary_monster - assigned to an hash with the following key/values
 #   - name (string)
 #   - smell (string)
 #   - weight (integer)
 #   - citiesDestroyed (array)
 #   - luckyNumbers (array)
 #   - address (hash with following key/values: number , street , state, zip)
-
+scary_monster = {
+  name: "Grendel",
+  smell: "Horrendous",
+  weight: 950,
+  citiesDestroyed: ["Copenhagen", "Oslo", "Stockholm"],
+  luckyNumbers: [7, 8, 9],
+  address: {
+    number: 999,
+    street: "Beowulf Lane",
+    state: "Michigan",
+    zip: 49855
+  }
+}
 
 # Create a new class called SuperHero
 # - Your class should have the following DYNAMIC values
-#   - name 
+#   - name
 #   - super_power
-#   - age 
+#   - age
 # - Your class should have the following STATIC values
 #   - arch_nemesis, assigned to "The Syntax Error"
 #   - power_level = 100
-#   - energy_level = 50 
+#   - energy_level = 50
 
 # - Create the following class methods
 #   - say_name, should print the hero's name to the terminal
 #   - maximize_energy, should update the energy_level to 1000
 #   - gain_power, should take an argument of a number and INCREASE the power_level by that number
+class SuperHero
+  attr_accessor :name, :super_power, :age
+
+  def initialize
+    @name = name
+    @super_power = super_power
+    @age = age
+    @arch_nemesis = "The Syntax Error"
+    @power_level = 100
+    @energy_level = 50
+
+  end
+
+  def say_name
+    puts "#{name}"
+  end
+
+  def maximize_energy
+    energy_level = 1000
+  end
+
+  def gain_power(number)
+    @power_level = @power_level * number
+  end
+end
 
 # - Create 2 instances of your SuperHero class
+codeman = SuperHero.new
```

Comment:
I think you want to create your `SuperHero` instance with 3 arguments passed in here, like so:

```
codeman = SuperHero.new('CodeMan', 'Debugging', 35)
```